installer/windows/installer/install.cmd

```diff
@@ -0,0 +1,7 @@
+@echo off
```

Comment:
this is a comment to the .keep file - why is that file here ?

Reply:
ah its for osx. never mind then.

Reply:
Like I was able to develop Protocol Handler for Linux and Windows I have to do for mac as well.
to maintain directory structure on github I have that keep file.